src/compiler/checker.ts

```diff
@@ -295,6 +295,15 @@ namespace ts {
             getAccessibleSymbolChain,
             getTypePredicateOfSignature,
             resolveExternalModuleSymbol,
+            tryGetThisTypeAt: node => {
+                node = getParseTreeNode(node);
+                return node && tryGetThisTypeAt(node);
+            },
+            isMemberSymbol: symbol =>
+                symbol.flags & SymbolFlags.ClassMember

```

Comment:
Asked @sandersn and @weswigham and they didn't know why these are properties, but if I try to change `undefined` to a variable I can't build the compiler.

Reply:
seems that this should be in utilities instead on the checker. nothing here is about the checker internal state, but rather how we mark symbols.